### PR TITLE
#88: Implement inline tables

### PR DESCRIPTION
**Pull request by ShevtsovY**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
   },
   "scripts": {
     "build": "./scripts/build",
+    "clean": "rm -rf /tmp/.yarn-bazel-cache/ && bazel clean",
```
Comment (lewish) [inline package.json]: You shouldn't need this anymore, I fixed the underlying issue

Comment (ShevtsovY) [reply package.json]: ok, I will remove it

File: core/utils.ts [hunk 2]
```diff
@@ -220,6 +232,18 @@ export function validate(compiledGraph: protos.ICompiledGraph): protos.IGraphErr
         });
       });
     }
+
+    // ignored properties in tables
+    if (!!ignoredProps[node.type]) {
+      ignoredProps[node.type].forEach(key => {
+        if (isPropertyExist(node[key])) {
```
Comment (BenBirt) [inline core/utils.ts]: is this `node.hasOwnProperty(key)`?

File: core/utils.ts [hunk 3]
```diff
@@ -220,6 +232,18 @@ export function validate(compiledGraph: protos.ICompiledGraph): protos.IGraphErr
         });
       });
     }
+
+    // ignored properties in tables
+    if (!!ignoredProps[node.type]) {
+      ignoredProps[node.type].forEach(key => {
```
Comment (BenBirt) [inline core/utils.ts]: `ignoredProp` instead of `key` please

Comment (ShevtsovY) [reply core/utils.ts]: ok

File: core/utils.ts (hunk 2 above)
Comment (ShevtsovY) [reply core/utils.ts]: not quite clear what you mean?! in this function we check for empty arrays and empty objects...

**Pull request merged**